crates/attestation-verification/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "attestation-verification"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
+anyhow = "1.0"
+async-trait = "0.1"
+clap = { version = "4.5", features = ["derive", "string", "env"] }
```

Comment:
clap doesn't belong here, it's a verifier specific one. Possibly serde_json doesn't either, I'm not sure